### ISSUE TITLE
FR: automatically strip every occurence of $pwd

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [paul-gauthier/aider#1167]
### Issue

Hi,

I'm quite privacy paranoid and when I'm using an LLM I like to avoid giving away some information as much as possible. In some situation, file paths can contain quite a bit of information (company name, projects, user name etc) and is in the large majority of cases not needed.

Those paths appear in my case when I'm using /run commands and get an error trace. In that case I think that the paths to the current directory could automatically be replaced by `~` or `.` on every LLM input without causing issues..

So basically my feature request is this: can you consider automatically applying a dumb `input_text. = input_text.replace(str(Path.cwd), ".")` after a /run command?

Just in case I guess you could add an env variable to disable it for the fringe cases of coding about filepaths I guess?
